packages/utils/src/__tests__/period/period/period.test.js

```diff
@@ -0,0 +1,246 @@
+      const testData = [
+        ['YEAR', YEAR],
+        ['quaRter', QUARTER],
+        ['mOnTh', MONTH],
+        ['wEEk', WEEK],
+        ['day', DAY],
+      ];
+
+      testData.forEach(([periodType, expected]) => {
+        expect(parsePeriodType(periodType)).toBe(expected);
+      });
+    });
+
+    it('should throw an error for a wrong input', () => {
+      const wrongValues = ['random', false, undefined, null, '201701', { MONTH: 'MONTH' }];
+      wrongValues.map(wrongValue =>
+        expect(() => parsePeriodType(wrongValue)).toThrowError('Period type'),
+      );
+    });
+  });
+
+  describe('momentToPeriod', () => {
+    const momentStub = { format: args => args };
+    const testData = [
+      ['year', YEAR, 'YYYY'],
+      ['quarter', QUARTER, 'YYYY[Q]Q'],
+      ['month', MONTH, 'YYYYMM'],
+      ['week', WEEK, 'GGGG[W]WW'],
+      ['day', DAY, 'YYYYMMDD'],
+    ];
+
+    it.each(testData)('%s', (_, periodType, expected) => {
+      expect(momentToPeriod(momentStub, periodType)).toBe(expected);
+    });
+  });
+
+  describe('dateStringToPeriod', () => {
+    const testData = [
+      ['should use `day` by default', '2020-02-15', undefined, '20200215'],
+      ['should convert compatible date formats: date', '2020-02-15', undefined, '20200215'],
+      [
+        'should convert compatible date formats - date & time',
+        '2020-02-15 10:18:00',
+        undefined,
+        '20200215',
+      ],
+      ['year', '2020-02-15', YEAR, '2020'],
+      ['quarter', '2020-02-15', QUARTER, '2020Q1'],
+      ['quarter', '2020-04-01', QUARTER, '2020Q2'],
+      ['month', '2020-02-15', MONTH, '202002'],
+      ['week', '2020-02-15', WEEK, '2020W07'],
+      ['day', '2020-02-15', DAY, '20200215'],
+    ];
+
+    it.each(testData)('%s', (_, date, periodType, expected) => {
+      expect(dateStringToPeriod(date, periodType)).toBe(expected);
+    });
+  });
+
+  describe('periodToTimestamp', () => {
+    const testData = [
+      ['year', '2016', 1451606400000],
+      ['quarter', '2016Q1', 1451606400000],
+      ['month', '201602', 1454284800000],
+      ['week', '2016W06', 1454889600000],
+      ['day', '20160229', 1456704000000],
+    ];
+
+    it.each(testData)('%s', (_, period, expected) => {
+      expect(periodToTimestamp(period)).toBe(expected);
+    });
+  });
+
+  describe('getCurrentPeriod', () => {
+    const currentDateStub = '2020-12-27';
+
+    beforeAll(() => {
+      MockDate.set(currentDateStub);
+    });
+
+    afterAll(() => {
+      MockDate.reset();
+    });
+
+    const testData = [
+      ['year', YEAR, '2020'],
+      ['quarter', QUARTER, '2020Q4'],
+      ['month', MONTH, '202012'],
+      ['week', WEEK, '2020W52'],
+      ['day', DAY, '20201227'],
+    ];
+
+    it.each(testData)('%s', (_, periodType, expected) => {
+      expect(getCurrentPeriod(periodType)).toBe(expected);
+    });
+  });
+
+  describe('periodToDisplayString', testPeriodToDisplayString);
```

Comment:
And here you can see an example of how we include the extracted tests in the main file. Those are the only changes in this commit, everything else is the same!